01_Tutorial/01_Basics.md

```diff
@@ -165,7 +165,7 @@ for now, we include this line to disable them.
 - `repo: './ipfs'` designates the path of the repo in Node.js only. In the browser, you can actually remove this line. The
 default setting is a folder called `.jsipfs` in your home directory. You will see why we choose this specific location for the
 folder later.
-- `EXPERIMENTAL: { pubsub: true }` enables [IPFS pubsub](https://blog.ipfs.io/25-pubsub/), which is a method of communicating between nodes and **is required for OrbitDB usage**, despite whether or not we are connected to other peers.
+- `EXPERIMENTAL: { ipnsPubsub: true }` enables [IPFS pubsub](https://blog.ipfs.io/25-pubsub/), which is a method of communicating between nodes and **is required for OrbitDB usage**, despite whether or not we are connected to other peers.
```

Comment:
Perhaps this could be removed and a note added somewhere else, about how OrbitDB uses PubSub.
(Maybe in the Introduction).